packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -11,22 +11,22 @@ export const sliderClassNames: SlotClassNames<SliderSlots> = {
   input: 'fui-Slider__input',
 };
 
-// Internal CSS variables. Private until we want to commit to them
```

Comment:
I see you removed this comment about keeping the variables private, but they are not exported from `react-components` is that intentional ?

Reply:
I just removed the comment for housekeeping. I didn't feel it was necessary to have it. But yes, they are not exported because they are only used internally.